sdk/communication/communication-jobrouter/review/communication-jobrouter.api.md

```diff
@@ -0,0 +1,693 @@
+    constructor(connectionString: string, routerClientOptions?: RouterClientOptions);
+    constructor(endpoint: string, credential: KeyCredential, routerClientOptions?: RouterClientOptions);
+    constructor(endpoint: string, credential: TokenCredential, routerClientOptions?: RouterClientOptions);
+    constructor(endpoint: string, credential: CommunicationTokenCredential, routerClientOptions?: RouterClientOptions);
+    // Warning: (ae-forgotten-export) The symbol "AcceptJobOfferResult" needs to be exported by the entry point index.d.ts
+    acceptJobOffer(workerId: string, offerId: string): Promise<AcceptJobOfferResult>;
+    cancelJob(jobId: string, options?: CancelJobOptions): Promise<JobRouterCancelJobActionResponse>;
+    closeJob(jobId: string, assignmentId: string, options?: CloseJobOptions): Promise<JobRouterCloseJobActionResponse>;
+    completeJob(jobId: string, assignmentId: string, options?: CompleteJobOptions): Promise<JobRouterCompleteJobActionResponse>;
+    createJob(jobId: string, options?: CreateJobOptions): Promise<RouterJob>;
+    createWorker(workerId: string, options?: CreateWorkerOptions): Promise<RouterWorker>;
+    declineJobOffer(workerId: string, offerId: string): Promise<JobRouterDeclineJobActionResponse>;
+    deleteJob(jobId: string): Promise<void>;
+    deleteWorker(workerId: string): Promise<void>;
+    deregisterWorker(workerId: string): Promise<RouterWorker>;
+    getJob(jobId: string): Promise<RouterJob>;
+    getQueuePosition(jobId: string): Promise<JobPositionDetails>;
+    getWorker(workerId: string): Promise<RouterWorker>;
+    listJobs(options?: ListJobsOptions): PagedAsyncIterableIterator<RouterJobItem>;
+    listWorkers(options?: ListWorkersOptions): PagedAsyncIterableIterator<RouterWorkerItem>;
+    // Warning: (ae-forgotten-export) The symbol "JobRouterReclassifyJobActionResponse" needs to be exported by the entry point index.d.ts
+    reclassifyJob(jobId: string, options?: ReclassifyJobOptions): Promise<JobRouterReclassifyJobActionResponse>;
+    registerWorker(workerId: string): Promise<RouterWorker>;
+    // Warning: (ae-forgotten-export) The symbol "JobRouterUnassignJobActionResponse" needs to be exported by the entry point index.d.ts
+    unassignJob(jobId: string, assignmentId: string): Promise<JobRouterUnassignJobActionResponse>;
+    updateJob(jobId: string, options?: UpdateJobOptions): Promise<RouterJob>;
+    updateWorker(workerId: string, options?: UpdateWorkerOptions): Promise<RouterWorker>;
+}
+
+// @public
+export interface RouterClientOptions extends PipelineOptions {
+    headers?: {
+        [propertyName: string]: any;
+    };
+}
+
+// @public
+export interface RouterJob {
+    readonly assignments?: {
+        [propertyName: string]: JobAssignment;
+    };
+    readonly attachedWorkerSelectors?: WorkerSelector[];
+    channelId?: string;
+    channelReference?: string;
+    classificationPolicyId?: string;
+    dispositionCode?: string;
+    readonly enqueueTimeUtc?: Date;
```

Comment:
It would be a large change to rename it on server side, I will keep it for now. I can create a task to rename it in future.

Reply:
We don't need to rename this in the server, we can update it in the swagger or add a swagger transform. Also I would think this may be better named `enqueuedOn`?